container/split.go

```diff
@@ -1,30 +1,316 @@
 package container
 
 import (
+	"image/color"
+
 	"fyne.io/fyne"
+	"fyne.io/fyne/canvas"
+	"fyne.io/fyne/driver/desktop"
+	"fyne.io/fyne/theme"
 	"fyne.io/fyne/widget"
 )
 
+// Declare conformity with CanvasObject interface
+var _ fyne.CanvasObject = (*Split)(nil)
+
 // Split defines a container whose size is split between two children.
-//
-// Since: 1.4
-type Split = widget.SplitContainer
+type Split struct {
+	widget.BaseWidget
+	Offset     float64
+	Horizontal bool
+	Leading    fyne.CanvasObject
+	Trailing   fyne.CanvasObject
+}
 
 // NewHSplit creates a horizontally arranged container with the specified leading and trailing elements.
 // A vertical split bar that can be dragged will be added between the elements.
-//
-// Since: 1.4
```

Comment:
Why remove these annotations?

Reply:
Apologies, that was a mistake, good catch